src/codegen/llvm/codegen_llvm.cc

```diff
@@ -962,7 +962,42 @@ llvm::Value* CodeGenLLVM::VisitExpr_(const Broadcast* op) {
 }
 
 void CodeGenLLVM::VisitStmt_(const Store* op) {
-  CHECK(is_one(op->predicate));
+  // CHECK(is_one(op->predicate));
+
+  // mask store
```

Comment:
>// mask store [](start = 2, length = 13)

please add some more explanation about this branch. Why check !is_one? why mask is needed? what's the difference between lanes() == 1 and != 1?